packages/firebase_crashlytics/darwin/Classes/FirebaseCrashlyticsPlugin.m

```diff
@@ -87,22 +82,15 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
       reason = [NSString stringWithFormat:@"thrown %@", context];
     }
 
-    [[Crashlytics sharedInstance] recordCustomExceptionName:call.arguments[@"exception"]
-                                                     reason:reason
-                                                 frameArray:frames];
+    // TODO: This is probably incorrect!
+    [[FIRCrashlytics crashlytics] recordError:call.arguments[@"exception"]]
```

Comment:
I don't know how to handle this, since the new SDK now requires to provide the exception directly instead of separating it into name, reason, and stacktrace...

Reply:
https://developer.apple.com/documentation/foundation/nsexception/1569530-exceptionwithname